torch/onnx/symbolic_opset9.py

```diff
@@ -2225,14 +2226,33 @@ def layer_norm(g, input, normalized_shape, weight, bias, eps, cudnn_enable):
     variance = g.op("ReduceMean", pow(g, numerator, two_cst), axes_i=axes)
     denominator = sqrt(g, add(g, variance, eps_cst))
 
-    layer_norm = g.op("Div", numerator, denominator)
+    normalized = g.op("Div", numerator, denominator)
 
     if not (weight is None or symbolic_helper._is_none(weight)):
-        layer_norm = mul(g, layer_norm, weight)
+        normalized = mul(g, normalized, weight)
     if not (bias is None or symbolic_helper._is_none(bias)):
-        layer_norm = add(g, layer_norm, bias)
+        normalized = add(g, normalized, bias)
+
+    if return_mean_rstd:
+        rdenominator = reciprocal(g, denominator)
+        return normalized, mean, rdenominator
+    else:
+        return normalized, None, None
+
 
-    return layer_norm
+@symbolic_helper.parse_args("v", "is", "v", "v", "f")
+def native_layer_norm(g, input, normalized_shape, weight, bias, eps):
+    return _layer_norm_shared(
+        g, input, normalized_shape, weight, bias, eps, False, True
+    )
+
+
+@symbolic_helper.parse_args("v", "is", "v", "v", "f", "i")
+def layer_norm(g, input, normalized_shape, weight, bias, eps, cudnn_enable):
+    normalized, mean, rstd = _layer_norm_shared(
```

Comment:
Suggest for unused variables:
```python
    normalized, _, _= _layer_norm_shared(
        g, input, normalized_shape, weight, bias, eps, cudnn_enable, False
    )
```

Reply:
Ok.

Reply:
Could you also do so for other PRs you submitted? Thanks a lot!

Reply:
Ok.